bedrock/mozorg/templates/mozorg/moss/index.html

```diff
@@ -48,6 +48,13 @@ <h2 class="moss-section-heading">{{ _('About MOSS') }}</h2>
     resources on a vibrant regional open source community - in this case,
     in India.
   {% endtrans %}
+  </p>
+
+    <p class="moss-copy">
+  {% trans %}
+    MOSS is an awards program of the Mozilla Corporation, which is ultimately responsible for selecting and
```

Comment:
Because we are adding new text to a page that has already been translated we need to wrap the new content in a `l10n_has_tag`. Make sure you include the `<p>` tags inside the if statement.

 There's a [code example in the docs](http://bedrock.readthedocs.io/en/latest/l10n.html#staging-copy-changes).

Let me know if that looks like too much coding too ;)